pkgs/applications/editors/vscode/extensions/ms-dotnettools.csharp/default.nix

```
@@ -89,6 +89,11 @@ vscode-utils.buildVscodeMarketplaceExtension rec {
       -E -e 's/(this\._pipePath=[a-zA-Z0-9_]+\.join\()([a-zA-Z0-9_]+\.getExtensionPath\(\)[^,]*,)/\1require("os").tmpdir(), "'"$ext_unique_id"'"\+/g' \
       "$PWD/dist/extension.js"
 
+    # Fix reference to uname
+    sed -i \
+      -E -e 's_uname -m_${coreutils}/bin/uname -m_g' \
+      "$PWD/dist/extension.js"
```

```suggestion
      dist/extension.js
```
This would be the same

---

I'll fix this on the next PR to update the extension (along with replacing `sed` with `substituteInPlace`).

Version 1.25.5 is already out but it seems to be a pre-release one so I've kept it out of the update.